src/js/jsx/sections/layers/LayerGroup.jsx

```diff
@@ -68,7 +72,25 @@ define(function (require, exports, module) {
             /**
              * @type {Boolean}
              */
-            disabled: React.PropTypes.bool
+            disabled: React.PropTypes.bool,
+
+            /**
+             * True if the LayerGroup should render collapsed layers. This is controlled by the 
+             * root LayerGroup.
+             * 
+             * @type {Boolean}
+             */
+            renderCollapsedLayers: React.PropTypes.bool
+        },
+        
+        componentDidMount: function () {
+            if (!this.props.parentLayer) {
+                this.renderCollapsedLayers = true;
+
+                window.setTimeout(function () {
+                    this.forceUpdate.bind(this, null);
```

Comment:
OK, I see now that this is the "wait a bit and then render everything" optimization, which I didn't get before. The comment above makes sense, I just didn't put it together. It might still be nice to have a one-liner at this if statement just to help dummies like me :) But, what still doesn't make sense is the timeout function, which just evaluates `this.forceUpdate.bind(this, null)`. This expression evaluates to a bound function, but it does not call the function or save a reference to it. Should this be `this.forceUpdate.call(this, null)` instead?